node-data/src/message.rs

```diff
@@ -411,35 +397,13 @@ impl Header {
         Status::Past
     }
 
-    pub fn verify_signature(
-        &self,
-        signature: &[u8; 48],
-    ) -> Result<(), dusk_bls12_381_sign::Error> {
-        let sig = dusk_bls12_381_sign::Signature::from_bytes(signature)?;
-
-        dusk_bls12_381_sign::APK::from(self.pubkey_bls.inner()).verify(
-            &sig,
-            marshal_signable_vote(
-                self.round,
-                self.get_step(),
-                &self.block_hash,
-            )
-            .bytes(),
-        )
-    }
-
-    pub fn sign(
-        &self,
-        sk: &dusk_bls12_381_sign::SecretKey,
-        pk: &dusk_bls12_381_sign::PublicKey,
-    ) -> [u8; 48] {
-        let msg = marshal_signable_vote(
-            self.round,
-            self.get_step(),
-            &self.block_hash,
-        );
+    pub fn signable(&self) -> Vec<u8> {
+        let mut buf = vec![];
+        buf.extend_from_slice(&self.round.to_le_bytes());
```

Comment:
Does signing `round` have any function?

Reply:
We rely on the "round" when detecting out of sync, so for now I would leave it in the signed fields